### ISSUE TITLE
output padding different to zero in embedding layer

### ISSUE THREAD

**Issue by akamnev** [huggingface/transformers#2944]
# 🐛 Bug
On embedding layer, the token corresponding to the padding does not return 0.
## Information

Model I am using (Bert):

Language I am using the model on (English):

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [x] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)

## To reproduce

```python
import torch
from transformers.tokenization_bert import BertTokenizer
from transformers.modeling_bert import BertEmbeddings

tokenizer = BertTokenizer.from_pretrained('bert-base-cased')

class Config:
    vocab_size = tokenizer.vocab_size
    hidden_size = 768
    max_position_embeddings = 512
    type_vocab_size = 2
    hidden_dropout_prob = 0.1
    layer_norm_eps = 1e-12

max_length = 10
sentence = "I eat a green apple"
tokens = tokenizer.encode(sentence)
tokens += [tokenizer.pad_token_id] * (max_length - len(tokens))
print(tokens)

embedding = BertEmbeddings(Config)
input_ids = torch.tensor([tokens])
emb = embedding(input_ids)

print(emb[0][-1])  # the last step should return 0 tensor
```
## Expected behavior
I expect to get a zero output tensor

## Environment info     
- `transformers` version: 2.3.0
- Platform: Ubuntu
- Python version: 3.7
- PyTorch version (GPU?): 1.3.1 (CPU)
- Tensorflow version (GPU?): no
- Using GPU in script?: no
- Using distributed or parallel set-up in script?: no

**Comment by BramVanroy** [comment]
Why? Typically, padding tokens are ignored in the model by the use of an attention mask. I don't understand why you want to only get the output of the embedding layer. (Perhaps you are confused with the good ol' word2vec embedding models, but you cannot/should not extract features from the embedding layer in a LM.)
